groovy-language-server/src/main/java/com/palantir/ls/groovy/GroovyLanguageServerConfig.java

```diff
@@ -31,13 +32,25 @@
 
 public final class GroovyLanguageServerConfig implements LanguageServerConfig {
```

Comment:
Can we rename this class to `GroovyLanguageServerSharedObjects` or something like that
Same with the interface